plugins/operators/__init__.py

```diff
@@ -1416,6 +1416,202 @@ def execute(self, ctx):
         ctx.dataset.create_index(field_name, unique=unique, wait=False)
 
 
+class ConvertIndexToUnique(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="convert_index_to_unique",  # Must match what's in fiftyone.yml
+            label="Convert an existing index to unique",
```

Comment:
We have an existing convention of the `label` exactly matching the operator `name` when converted to sentence case. So here we'd have:

```py
label="Convert index to unique"
```